src/pages/workspaces/workspace/useWorkspace.test.ts

```diff
@@ -188,11 +190,16 @@ describe('useActiveWorkspace', () => {
     asMockedFn(Ajax).mockImplementation(() => mockAjax as AjaxContract)
 
     workspaceStore.set(initializedGoogleWorkspace)
-    // Calling to get the bucket location fails, default options remain
-    const expectedStorageDetails = _.merge(defaultGoogleBucketOptions, defaultAzureStorageOptions)
+    // Calling to get the bucket location fails, default options remain except for indication of error state.
+    const expectedStorageDetails = _.mergeAll([
+      defaultGoogleBucketOptions,
+      { fetchedGoogleBucketLocation: 'ERROR' },
+      defaultAzureStorageOptions
+    ])
 
     // Act
-    const { result } = renderHook(() => useWorkspace('testNamespace', 'testName'))
+    const { result, waitForNextUpdate } = renderHook(() => useWorkspace('testNamespace', 'testName'))
+    await waitForNextUpdate() // For the bucket location call
```

Comment:
This was actually a bug in the test (we were never making the bucket location call because we didn't do `waitForNextUpdate`). The introduction of `fetchedGoogeBucketLocation` revealed it.